ncard_app/templates/authenticate/login.html

```diff
@@ -0,0 +1,20 @@
+{% extends 'events/base.html' %}
+
+{% block title %}Login{% endblock %}
+
+{% block content%}
+    <h1>Login</h1>
+    <br>
+    <form action="{% url 'login' %}" method=POST>
+        {% csrf_token %}
+        <div class="mb-3">
+            <label for="exampleInputEmail1" class="form-label">Username</label>
```

Comment:
The 'for' attribute on labels should match the 'id' attribute of the input being labelled.